src/PolykeyClient.ts

```diff
@@ -1,18 +1,20 @@
 import type { FileSystem } from './types';
-import type { StreamFactory } from './rpc/types';
+import type { StreamFactory, IdGen } from '@matrixai/rpc';
 import path from 'path';
 import Logger from '@matrixai/logger';
 import { CreateDestroyStartStop } from '@matrixai/async-init/dist/CreateDestroyStartStop';
-import RPCClient from './rpc/RPCClient';
-import * as rpcUtilsMiddleware from './rpc/utils/middleware';
+import { RPCClient } from '@matrixai/rpc';
+import { middleware as rpcUtilsMiddleware } from '@matrixai/rpc';
 import * as clientUtilsMiddleware from './client/utils/middleware';
 import { Session } from './sessions';
 import * as utils from './utils';
 import * as errors from './errors';
 import * as events from './events';
 import config from './config';
 import { clientManifest } from './client/handlers/clientManifest';
+import * as networkUtils from './network/utils';
 
+let idGen: IdGen;
```

Comment:
Why is `idGen` defined outside the entire class?

Reply:
I have no idea, that should not be done.

Reply:
Yea, also I'm pretty the `idGen` is now optional. So for now just get rid of it.

Reply:
By default removing `idGen` should be making it so that `id` field is always just `null`. As that is allowed in the JSON RPC spec.

Reply:
In the future we can add an auto-increment. Or... even something useful for tracing.

Reply:
Removed.